addons/compat_gm/config.cpp

```diff
@@ -0,0 +1,22 @@
+#include "script_component.hpp"
+#include "\z\ace\addons\csw\script_config_macros_csw.hpp"
```

Comment:
This should be inside `script_component.hpp`.

Reply:
fixed in 4aa075267f70e83007da411222c0ee0a68bb5cb8